decidim-core/lib/decidim/core/engine.rb

```diff
@@ -643,13 +652,11 @@ class Engine < ::Rails::Engine
 
         Decidim.register_social_share_service("Facebook") do |service|
           service.icon = "facebook-circle-line"
-          service.icon_color = "#1877f2"
           service.share_uri = "http://www.facebook.com/sharer/sharer.php?u=%{url}"
         end
 
         Decidim.register_social_share_service("Google Bookmark") do |service|
           service.icon = "google-line"
-          service.icon_color = "#4285f4"
```

Comment:
As we're dropping the icon_color attribute, we should remove it from the docs too: ./docs/modules/customize/pages/social_shares.adoc

Reply:
fixed in def6b6e67b